log/log.go

```diff
@@ -58,3 +60,32 @@ type LoggerFunc func(...interface{}) error
 func (f LoggerFunc) Log(keyvals ...interface{}) error {
 	return f(keyvals...)
 }
+
+// SwapLogger wraps another logger that may be safely replaced while other
+// goroutines use the SwapLogger concurrently.
```

Comment:
I'm not opposed to adding that, but I wonder if it would be better to put it in the package docs if we include other advice on how to best use this package. Maybe it belongs in both. Thoughts?